assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-cta.scss

```diff
@@ -19,13 +19,27 @@
 .googlesitekit-plugin {
 
 	.googlesitekit-idea-hub__dashboard-cta {
+		align-items: center;
 		background-color: $c-white;
-		display: flex;
+		height: 100%;
+		padding: $grid-gap-desktop;
```

Comment:
We probably shouldn't be using desktop values in mobile – I'm not even sure if we want added padding in mobile (not shown in the design). I suppose it would be more consistent with the design to keep it, but let's use the mobile/phone value here instead.

Reply:
@johnPhillips Since we used the `Widget` component to wrap the content, we have the `googlesitekit-widget__body` class name which has `16px` and `24px` padding on small screens and large screens respectively. Then we have our `.googlesitekit-idea-hub__dashboard-cta` class which further adds `48px` padding which amounts to a total of `24 + 48 = 72px` on large screens and `16 + 48 = 64px` on small screens. So let's remove the padding here, relying on the `16px` padding on small screens from the `googlesitekit-widget__body` class.

Reply:
Makes sense, thanks.